### DISCUSSION TITLE
Please re-ad (SFW channels) ADULT SWIM EAST & ADULT SWIM WEST

### DISCUSSION THREAD

**Discussion by Raygeee1** [iptv-org/community#1156] [Q&A]
These are not adult Channels but it seems were erroneously deleted with other NSFW channels. This is a cartoon network program that airs at night in certain markets and was my goto. Can you please re add these? Thank you!!

**Comment by BellezaEmporium** [comment]
It was DMCAd by WBD a long time ago.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/16839

**Reply by Raygeee1** [reply]
Thanks
